SpatialOS/spatialos.json

```diff
@@ -1,12 +1,12 @@
 {
   "configurationVersion": "0.1",
   "projectName": "your_project_name_here",
-  "schemaDescriptor": "./schema/bin/schema.descriptor",
+  "schemaDescriptor": "./schema_out/bin/schema.descriptor",
```

Comment:
Sorry for the last minute request, can the schema descriptor be moved out of the bin folder?

Reply:
done